Sources/RegexBenchmark/BenchmarkResults.swift

```diff
@@ -263,6 +290,27 @@ struct SuiteResult {
 }
 
 extension SuiteResult: Codable {
+  func saveCSV(to url: URL) throws {
+    var output: [(name: String, result: BenchmarkResult)] = []
+    for key in results.keys {
+      output.append((key, results[key]!))
+    }
+    output.sort {
+      $0.name < $1.name
+    }
+    var contents = """
+    name,\
```

Comment:
You can escape the entire field in double double quotes, e.g. `""field 1""`

Reply:
I'm going to not do so now, as I think it will put the quote in the field when opened in a spreadsheet. We can add that later or just have a policy that names not include commas.